app/src/views/features/sessions/SessionViewer/SessionPropertiesPanel.tsx

```diff
@@ -28,6 +28,7 @@ const SessionPropertiesPanel: React.FC<Props> = ({ getCurrentTimeOffset }) => {
   const startTimeOffset = useSelector(getSessionRecordingStartTimeOffset);
   const isReadOnly = useSelector(getIsReadOnly);
   const sessionDescription = useSelector(getSessionRecordingDescription);
+  const [currentSavedDescription, setCurrentSavedDescription] = useState(sessionDescription);
```

Comment:
Instead of `state` let's use a `ref`
```diff
- const [currentSavedDescription, setCurrentSavedDescription] = useState(sessionDescription);
+ const savedDescriptionRef = useRef(sessionDescription);
```